test/testutils.js

```diff
@@ -94,4 +94,54 @@ testUtils.checkSetup = function (content, options, target) {
 	return [node.actualNode, options, node];
 };
 
+
+/**
+ * Create check arguments with Shadow DOM
+ *
+ * @param Node|String 	Stuff to go into the fixture (html or node)
+ * @param Node|String 	Stuff to go into the shadow boundary (html or node)
+ * @param Object  			Options argument for the check (optional, default: {})
+ * @param String  			Target for the check, CSS selector (default: '#target')
+ * @return Array
+ */
+testUtils.shadowCheckSetup = function (content, shadowContent, options, target) {
```

Comment:
Why is this here?

Reply:
I created it for them in a separate branch, there was a lot of boilerplate for shadow DOM tests that I wanted to smooth out.